src/docs/drizzle/react/react-integration.md

```diff
@@ -4,145 +4,155 @@ layout: docs.hbs
 ---
 # React Integration
 
-`drizzle-react` is the official way to integrate Drizzle with your React dapp.
+`@drizzle/react-plugin` is the official way to integrate Drizzle with your React dapp.
 
-Check out the [Drizzle Truffle Box](https://github.com/truffle-box/drizzle-box) for a complete example (also featuring `drizzle-react-components`), or continue reading to create your own setup.
+Check out the [Drizzle Truffle Box](https://github.com/truffle-box/drizzle-box) for a complete example or continue reading to create your own setup.
+
+Also, check out `@drizzle/react-components` for Drizzle's [prebuilt react components](https://github.com/trufflesuite/drizzle/tree/develop/packages/react-components).
 
 ## Installation
 
-Install Drizzle React Components via npm:
+Install Drizzle React-Plugin via npm:
 ```bash
-npm install --save drizzle-react
+npm install @drizzle/react-plugin
 ```
 
-**Note**: `drizzle-react` requires Requires React 0.14 or greater. You'll also need the `drizzle` package, if it isn't already installed.
+**Note**: `@drizzle/react-plugin` requires Requires React v16.3+ for the Context API. You'll also need the `@drizzle/store` package, if it isn't already installed.
 
 ## Getting Started
 
 **Note**: Since Drizzle uses web3 1.0 and web sockets, be sure your development environment can support these.
```

Comment:
We shouldn't need this warning as @drizzle/store resolves `web3` which has matured since drizzle was initially written

Reply:
you're 100% right. thanks @cds-amal